components/public-api-server/pkg/apiv1/oidc_test.go

```diff
@@ -48,19 +49,26 @@ func TestOIDCService_CreateClientConfig(t *testing.T) {
 		require.Equal(t, connect.CodePermissionDenied, connect.CodeOf(err))
 	})
 
-	t.Run("feature flag enabled returns unimplemented", func(t *testing.T) {
+	t.Run("feature flag enabled returns created config", func(t *testing.T) {
 		serverMock, client := setupOIDCService(t, withOIDCFeatureEnabled)
 
 		serverMock.EXPECT().GetLoggedInUser(gomock.Any()).Return(user, nil)
 
-		_, err := client.CreateClientConfig(context.Background(), connect.NewRequest(&v1.CreateClientConfigRequest{}))
-		require.Error(t, err)
-		require.Equal(t, connect.CodeUnimplemented, connect.CodeOf(err))
+		response, err := client.CreateClientConfig(context.Background(), connect.NewRequest(&v1.CreateClientConfigRequest{
```

Comment:
Do you want to assert the right response was returned? There's a helper `requireProtoEqual` for that. It would help test the conversation for `iam.OIDCClientConfig` to `public-api.OIDCClientConfig`

Reply:
Good point. I'll have a look to test conversion, as they were PITA during manual testing.

The config is pinged back here by the stub implementation, though.

Reply:
Now I got what you meant. It's the `requireEqualProto` function, available for those kind of assertions. 
✔️